recipe/yum_requirements.txt

```diff
@@ -0,0 +1,5 @@
+java-1.8.0-openjdk-devel
+unzip
+zip
+zlib-devel
+rsync
```

Comment:
The build_pip_package script uses rsync to copy some files over into the whl file.  Without rsync the `bazel-bin/tensorflow/tools/pip_package/build_pip_package $SRC_DIR/tensorflow_pkg` command fails.